pypeman/plugin_mgr.py

```diff
@@ -0,0 +1,96 @@
+"""
```

Comment:
Did you consider to use https://pluggy.readthedocs.io/en/latest/ ? I think you should give a lo

Reply:
Hmmm, 

All energy is currently invested into the development of plug-in, that is urgently needed for production and that is already running in a validation environment, but still not feature complete.

All the code, that is specific to the way the plugins are added are already implemented.

The work is currently focused on adding features to the plugin, 
In other words the code, that is being written now will not change if I switched to (pluggy)

My suggestion would be to create a ticket (suggesting to switch hand written plugin manager logic to a pluggy logic) and do this perhaps in a 0.4.x or 0.5 release.

Your opinion

Reply:
Looks interesting. But will require time to deep dive and see how to use pluggy instead of the currently suggested system

Reply:
Ok for the ticket ;-)

Reply:
#151